client/src/pages/Landing.js

```diff
@@ -6,36 +6,36 @@ import { Route, Link } from "react-router-dom";
 
 import Ping from "./Ping";
 
-const landinPageStyle = theme => ({
+const landinPageStyle = (theme) => ({
```

Comment:
You can go ahead and delete this file, be sure to remove the reference to it in App.js

Reply:
cool